angular.json

```diff
@@ -130,7 +130,8 @@
           "builder": "@angular-devkit/build-angular:protractor",
           "options": {
             "protractorConfig": "e2e/protractor.conf.js",
-            "devServerTarget": "moh-sitereg:serve"
+            "devServerTarget": "moh-sitereg:serve",
+            "webdriverUpdate": false
```

Comment:
Can you look at configuring "update-config.json"  it is mentioned in the logs everytime we run e2e tests, and I think if we set it up it'll save time in the long run. We would potentially remove this `"webdriverUpdate": false` line, depending on update-config.json behaviour.

Google gave me these:
https://github.com/angular/webdriver-manager/issues/169

// examples there (named "config.json") seem super simple, just set version numbers.
https://medium.com/@cnishina/webdriver-manager-a-node-module-c33a820f5eb